includes/std/mem.pat

```diff
@@ -22,9 +22,12 @@ namespace std::mem {
     };
 
     fn find_sequence(u128 occurrence_index, auto ... bytes) {
-        return builtin::std::mem::find_sequence(occurrence_index, bytes);
+        return builtin::std::mem::find_sequence_in_range(occurrence_index, 0, 0, bytes);
```

Comment:
This function should take `std::mem::base_address()` and `std::mem::size()` in instead of just `0, 0` pretty sure.
Also don't forget to include `<std/mem.pat>` then

Reply:
Props to that, it would be much more readable.